### ISSUE TITLE
Lab 8 API Provision time

### ISSUE THREAD

**Issue by WigF1** [MicrosoftLearning/AZ-204-DevelopingSolutionsforMicrosoftAzure#66]
# Module: 08
#### Task: 02

In Lab 8, the estimated time is 45 minutes according to the lab manual. 

I have yet to provision an API in less than 30-35 minutes and it usually takes > 40 minutes to provision. This always seems to have been the case in Azure.

The LAK in Ex.2, Task 2 has students create a new API and under Step 6(v) there is a note that says "wait for the new API to finish being created".

Should we not set an expectation that this will be **at least** 30 minutes? Or consider having the resource group and API created first before performing Exercise 1?

**Comment by seesharprun** [comment]
Are you using the **Consumption** tier for API Managment?

We switched from the Developer tier to Consumption because of the exceedingly long time to provision a new service instance. The Consumption tier is serverless and advertised as having an **instantaneous** deployment time.

I just did a deployment to **East US** using the **Consumption** tier and my deployment completed in less than three minutes.
